commit 2cc2e3a2f6d1afd47cc076b89215ac41a0d41a92
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1336512887 -0500

    Fix kmix not autostarting in the user's session.
    This resolves bug report 503.

diff --git a/kmix/Makefile.am b/kmix/Makefile.am
index 5051363..d2da702 100644
--- a/kmix/Makefile.am
+++ b/kmix/Makefile.am
@@ -49,7 +49,7 @@ xdg_apps_DATA = kmix.desktop
 rcdir = $(kde_datadir)/kmix
 rc_DATA = kmixui.rc
 
-autostart_DATA = restore_kmix_volumes.desktop
+autostart_DATA = kmix.desktop restore_kmix_volumes.desktop
 autostartdir = $(datadir)/autostart
 
 lnkdir = $(kde_datadir)/kicker/applets
